vendor/engines/secure_rooms/config/locales/en.yml

```diff
@@ -55,6 +55,9 @@ en:
     attributes:
       user:
         card_number: Card Number
+      order_detail/occupancy:
+        exit_at: Time of Exit
+        actual_duration_mins: Duration
```

Comment:
these are also below under `secure_rooms/occupancy`, do they need to be under`order_detail/occupancy` as well for some lookup thing?